### ISSUE TITLE
关于数据集存放格式的询问

### ISSUE THREAD

**Issue by CurryaNa** [YubiaoYue/MedMamba#50]
请问在train.py文件里，我该如何放置我的数据集呢，以什么样的格式呢，是下边这样的格式吗

![Image](https://github.com/user-attachments/assets/8905f29c-e1ca-4cc1-9718-3e85fe55c41a)

**Comment by gudt-to-pku** [comment]
你需要下载数据集后自己写一个demo将其转为IamgeFolder，就是图中这种

**Comment by durian1023** [comment]
> 你需要下载数据集后自己写一个demo将其转为IamgeFolder，就是图中这种

谢谢你

**Comment by algwang** [comment]
> > 你需要下载数据集后自己写一个demo将其转为IamgeFolder，就是图中这种
> 
> 谢谢你

I would like to ask if we only have a training set and a validation set, or do we also need to add a test set?
